### ISSUE TITLE
CS50 database

### ISSUE THREAD

**Issue by sethwoodworth** [FinalsClub/djKarma#42]
pre-populate fall harvard courses with the CS50 db

We might want to hide courses from the big list view if they don't have classes yet. We need to think through that interaction.

**Comment by AndrewMagliozzi** [comment]
Indeed. Only courses with files to view should be listed in the courses view.

**Comment by sethwoodworth** [comment]
Uh oh. I have the code written to parse and store cs50 courses, but the api isn't working.  Their app seems to still work on an internal json-based api, but their old url structure returns nothing

http://courses.cs50.net/api/1.0/courses?output=php

**Comment by AndrewMagliozzi** [comment]
Bummer. We should have scraped that db when we had the chance.
